sdk/include/reactos/shlwapi_undoc.h

```diff
@@ -370,6 +370,9 @@ IContextMenu_Invoke(
     _In_ HWND hwnd,
     _In_ LPCSTR lpVerb,
     _In_ UINT uFlags);
+HRESULT WINAPI RunRegCommand(_In_opt_ HWND hWnd, _In_ HKEY hKey, _In_opt_ PCWSTR pszSubKey);
```

Comment:
```suggestion

HRESULT WINAPI RunRegCommand(_In_opt_ HWND hWnd, _In_ HKEY hKey, _In_opt_ PCWSTR pszSubKey);
```
Also ideally, prefer formatting these like the `IContextMenu_Invoke` function above.